kola-denylist.yaml

```
@@ -25,3 +25,24 @@
     - ppc64le
   streams:
     - rawhide
+- pattern: rpmostree.install-uninstall
+  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1515
+  snooze: 2023-07-12
+  arches:
+    - x86_64
```

Since we're only seeing this on azure WDYT of something like:

```suggestion
  platforms:
    - azure
```

---

Thanks for the suggestion. I updated the commit